Makefile

```diff
@@ -98,11 +92,21 @@ lint_module: $(FAILLINT) $(GOLANGCI_LINT) $(MISSPELL)
 	@cd $(DIR) && $(GOLANGCI_LINT) run
 	@./scripts/git-tree.sh
 
-# TODO(bwplotka): Move to buf.
-$(PROTOC):
+
+# For protoc naming matters.
+PROTOC_GEN_GO_CURRENT := $(TMP_GOPATH)/protoc-gen-go
+PROTOC_GEN_GO_GRPC_CURRENT := $(TMP_GOPATH)/protoc-gen-go-grpc
+PROTO_TEST_DIR := testing/testpb/v1
+
+.PHONY: proto
+proto: ## Generate testing protobufs
+proto: $(BUF) $(PROTOC_GEN_GO) $(PROTOC_GEN_GO_GRPC) $(PROTO_TEST_DIR)/test.proto
```

Comment:
Should we have two targets name `proto`? I'm not familiar with this Makefile pattern, if it is one.

Reply:
I can see this being confusing, but it's totally ok. It's to ensure `make help` works (and shows this command)